lib/AdapterLdap.php

```diff
@@ -226,6 +231,31 @@ public function getFacilitiesByEntityId($spEntityId)
         // TODO: Implement getEntityByEntityId() method.
     }
 
+    public function getFacilityByEntityId($spEntityId)
+    {
+        $facilities = $this->connector->searchForEntity(
```

Comment:
Using another name than $facilities (for example $facilityArray or $ldapResult) will be better.